MoltenVK/MoltenVK/Vulkan/mvk_datatypes.mm

```diff
@@ -32,111 +32,116 @@
 
 #pragma mark Pixel formats
 
-static MVKPixelFormats _platformPixelFormats;
+static std::unique_ptr<MVKPixelFormats> _platformPixelFormats;
+
+static MVKPixelFormats* getPlatformPixelFormats() {
+	if ( !_platformPixelFormats ) { _platformPixelFormats.reset(new MVKPixelFormats()); }
```

Comment:
Use std::make_unique C++14

https://github.com/AnthonyCalandra/modern-cpp-features#stdmake_unique

```suggestion
	if ( !_platformPixelFormats ) { _platformPixelFormats = std::make_unique<MVKPixelFormats>(); }
```

Reply:
I actually don't mind either way. Of the three justifications listed, two don't apply here. The type of the pointer is already specified, because it is a global object. And there aren't multiple calls that could throw an exception in that expression. As for the first, avoiding using `new` directly, this project uses `new` extensively.

Reply:
Agreed doesn't really matter. Just thought I would throw it out there.

Reply:
> Agreed doesn't really matter. Just thought I would throw it out there.

Excellent observation! I didn't consider C++14 options. 

But, I do prefer the existing form, as it explicitly matches the `new MVKObj()` form found elsewhere, and feels more like the activity telling the existing empty `unique_ptr` to "manage this now". 

Now, if only the function was called `set()`, instead of `reset()`, to match the corresponding `get()`. 😉